custom/icds_reports/tasks.py

```diff
@@ -29,7 +29,7 @@ def run_move_ucr_data_into_aggregation_tables_task(date=None):
 
 
 @serial_task('move-ucr-data-into-aggregate-tables', timeout=30 * 60, queue='background_queue')
-def move_ucr_data_into_aggregation_tables(date=None, intervals=3):
+def move_ucr_data_into_aggregation_tables(date=None, intervals=2):
```

Comment:
This could be an unfortunate consequence. May need to flag with the dashboard team to see how much historical data is required when writing new indicators

Reply:
Hmm.  It would be easy to iterate from -2 months -> +1 months if that's preferable.